ridgeback_control/launch/control.launch

```diff
@@ -1,14 +1,25 @@
 <launch>
-  <rosparam command="load" file="$(find ridgeback_control)/config/control.yaml" />
+  <arg name="robot_namespace" default="r0"/>
+  <!-- Generate the tf_prefix from the namespace -->
+  <arg name="tfpre" default="$(arg robot_namespace)_tf" />
+  <!-- Put value on the parameter server in the group's namespace -->
+  <param name="tf_prefix" value="$(arg robot_namespace)_tf" />
 
+  <rosparam command="load" file="$(find ridgeback_control)/config/control.yaml" />
+  <rosparam param="ridgeback_velocity_controller/base_frame_id" subst_value="True">$(arg tfpre)/base_link</rosparam>
+  <rosparam param="ridgeback_velocity_controller/odom_frame_id" subst_value="True">$(arg tfpre)/odom</rosparam>
+ 
   <group if="$(optenv RIDGEBACK_CONTROL_EXTRAS 0)" >
     <rosparam command="load" file="$(env RIDGEBACK_CONTROL_EXTRAS_PATH)" />
   </group>
 
   <node name="controller_spawner" pkg="controller_manager" type="spawner"
         args="ridgeback_joint_publisher ridgeback_velocity_controller" />
 
-  <node pkg="robot_localization" type="ekf_localization_node" name="ekf_localization" />
+  <!-- ekf localization temporarily disabled
```

Comment:
What's the trigger to enable it? (in other words, it was temporarily disabled for what reason, and when will it be reenabled?)